src/orange/construct/OrangeInput.hh

```diff
@@ -105,11 +105,15 @@ struct OrangeInput
 {
     std::vector<UnitInput> units;
 
+    // TODO: Calculate automatically in Shift by traversing the parent/daughter
+    // tree
+    size_type max_level = 20;
```

Comment:
Until we calculate this or set automatically, can you set it to 1? We don't want to blow up memory usage for our production test runs that have only a single level...

Reply:
Can I set it to 3? Because I have a test with 3 levels. Otherwise I will have to figure out how to calculate levels before this PR goes in.

Reply:
Oh sure. I didn't see any new tests but I forgot that the "initialization" test already existed. It might be a good thing to add to this PR though: just a simple update in the JSON input for now, and (though I can't find it) I thought that we calculated the max depth in the `Geometry_Builder.cc`...

Reply:
We probably want this functionality within Celeritas itself, not just Exnihilo, right? I.e. not just hard-coded in JSON. I am not sure what is going on with `max_level` Shift, but I don't think the functionality we want is actually in there now either. I think it's better to punt on this and I can address in the next PR.